network.tf

```diff
@@ -63,8 +63,7 @@ resource "google_compute_firewall" "istio_discovery_rule" {
   priority      = 1000
   direction     = "INGRESS"
   source_ranges = [var.master_ipv4_cidr_block]
-  // As we don't know the network tag, we allow for all
-  //target_tags   = ["gke-${var.project-id}-cluster-node-gke-28f667bc-node"]
+  target_tags   = [var.environment]
```

Comment:
Would this make some difference in an existing cluster created before this change?

Reply:
It depends on your node pool definition. 

If your node pool is tagged like the [one in this module](https://github.com/Devgurusio/terraform-gcp-gke-ecommerce/blob/master/nodes.tf#L50) this will only narrow the rule to only those nodes instead of the whole network. 
If not, this change will break the istio webhook as the master won't be able to talk to istio pods anymore

Reply:
I see, luckily I do have the tag you linked. Thanks!